<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_comp_7</class>
 <widget class="QDialog" name="fct_comp_7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sportifs d'un pays et un Ã©quipe</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>NumEpreuve</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_fct_7_pays"/>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Numero Inscrit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_fct_7_Eq"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_fct_7">
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table_fct_comp_7">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>70</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>numIn</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nomEp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>supprimer</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_fct_comp_7">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_fct_7</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_7</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_fct_7_pays</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_7</receiver>
   <slot>refreshBox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
  <slot>ok()</slot>
  <slot>refreshBox()</slot>
 </slots>
</ui>
